src/plone/app/multilingual/browser/interfaces.py

```diff
@@ -27,7 +28,8 @@ def make_relation_root_path(context):
 
     # Try to find the "closest" object in the target language
     current_object = context
-    target_language = context.REQUEST.get("language", None)
+    request = context.REQUEST if hasattr(context, "REQUEST") else getRequest()
+    target_language = request.get("language", None)
```

Comment:
I think `LANGUAGE` has to be uppercase. See https://github.com/plone/plone.api/blob/main/src/plone/api/portal.py#L391 and I suggest do use `getRequest` only instead of looking for `REQUEST` attribute on context.

Reply:
done `LANGUAGE` and used `getRequest` only.

Reply:
The problem is the function code that expects a content object anyway. So I've stepped back to just check if context is None or the REQUEST is None, this will avoid failures using `make_relation_root_path` in the control panel widgets.